deploy.cmd

```diff
@@ -0,0 +1,29 @@
+
+@echo off
+
+"tools\nuget\nuget.exe" "install" "FAKE.Core" "-OutputDirectory" "tools" "-ExcludeVersion" "-version" "4.4.2"
+
+:Build
+cls
+
+SET TARGET="CreatePackages"
+SET GITOWNER="octokit"
```

Comment:
I can see it's also used to find the right remote (I have both my fork and the upstream `octokit` repos tracked as remotes, so perhaps this needs to be two variables?